backendAPI/views.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.views.generic import TemplateView
+from django.http import JsonResponse
+from caffe_app.models import ModelExport
+# from django.shortcuts import render
+# from django.core import serializers
+
+
+# Create your views here.
+class login(TemplateView):
+    template_name = 'home.html'
+
+
+def checkLogin(request):
+    return JsonResponse({'result': request.user.is_authenticated()})
```

Comment:
As mentioned earlier, can you please follow the underscore style instead of camelCasing?